src/ochanticipy/config/countryconfig.py

```diff
@@ -7,37 +7,59 @@
 from ochanticipy.utils.io import parse_yaml
 
 
+class CustomLayerNames(BaseModel):
+    """Custom layer names for CodAB."""
+
+    name: str
+    hdx_resource: Optional[int]
```

Comment:
Overall this is a great implementation if we ever need it, however I would say since we don't have a situation where a country has both multiple resources and custom layer names, we can remove it for now just to keep it simple and avoid over engineering. And if it does come up then we have this code in our back pockets!